jcalcaben/pwa-studio-adobe-docs
# Apply updates from source template repository

## Description

Applies updates from the source template repository using the following command:

```
git cherry-pick 8e935799f193d5f03042ffa9723d43e662558505..c93127894054b28434386a861c03daf553af8bcc
```

## Related Issue

N/A

## Motivation and Context

Keeps this project up-to-date with the latest changes from the template repository.

## How Has This Been Tested?

1. Clean project cache
1. Run the dev server and production server 
1. Verify site builds

## Screenshots (if appropriate):

## Types of changes




## Checklist:
